eng/IndirectReferences.props

```diff
@@ -0,0 +1,3656 @@
+		<IndirectReference Include="Microsoft.AspNetCore.Cryptography.Internal" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.FileProviders.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Options" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Logging.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Configuration.Abstractions" KeepDuplicates="false" />
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.AspNetCore.Blazor'))">
+		<IndirectReference Include="Microsoft.AspNetCore.Components" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Primitives" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.DependencyInjection" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Components.Web" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Options" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Logging.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Components.Forms" KeepDuplicates="false" />
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.AspNetCore.Razor'))">
+		<IndirectReference Include="Microsoft.AspNetCore.Html.Abstractions" KeepDuplicates="false" />
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.AspNetCore.Antiforgery'))">
+		<IndirectReference Include="Microsoft.AspNetCore.WebUtilities" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Hosting.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Net.Http.Headers" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Cryptography.Internal" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Options" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Extensions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Configuration.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.DataProtection" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Extensions.Primitives" KeepDup
```

Comment:
I manually created these with references from Microsoft.AspNetCore and Microsoft.AspNetCore.Mvc. It would be nice to generate these too if possible.

Reply:
I can change the logic in my target to include sample/asset projects, will try that locally